server/container/cli/src/test/java/org/apache/james/cli/type/CmdTypeTest.java

```diff
@@ -139,6 +139,12 @@ public void lookupDeleteMailboxShouldReturnEnumValue() {
     }
 
     @Test
+    public void lookupImportEmlFileToMailboxShouldReturnEnumValue() {
+        assertThat(CmdType.lookup("ImportEmlFileToMailbox"))
```

Comment:
Maybe we can get the name of the command shorter... Maybe `importEml` is enough...

I 'm thinking to poor admins...